src/Bridge/Symfony/Bundle/Resources/config/nelmio_api_doc.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
you can ignore these in phpstan see what we did at https://github.com/api-platform/core/commit/bfca0eb9f064eff70ea570ba18c0326e6dd7291f#diff-6f19df6a6307a48db0940e6897591fb08776d2db8a6134737aa708defdb5c92dR19

Reply:
Done, thx :)